core/src/main/java/com/graphhopper/util/PathMerger.java

```diff
@@ -183,4 +182,53 @@ private void calcAscendDescend(final PathWrapper rsp, final PointList pointList)
         rsp.setAscend(ascendMeters);
         rsp.setDescend(descendMeters);
     }
-}
\ No newline at end of file
+
+    private String unconnectedListString(List<Path> paths) {
+        StringBuilder sb = new StringBuilder().append("[");
+
+        for (int n = 0; n < paths.size(); ++n) {
+            if (!paths.get(n).isFound())
+                sb.append(String.format("(%d,%d), ", n, n+1));
+        }
+
+        if (sb.length() < 2)
+            throw new IllegalArgumentException("list doesn't have unavailable paths");
```

Comment:
Maybe a better exception would be: "All provided paths are found" or "The paths don't contain an unfound path"?